src/EventStore.Client/EventStoreClientSettings.ConnectionString.cs

```diff
@@ -166,8 +169,20 @@ private static class ConnectionStringParser {
 				if (typedOptions.TryGetValue(ThrowOnAppendFailure, out object throwOnAppendFailure))
 					settings.OperationOptions.ThrowOnAppendFailure = (bool)throwOnAppendFailure;
 
-				if (typedOptions.TryGetValue(KeepAlive, out var keepAliveMs)) {
-					settings.ConnectivitySettings.KeepAlive = TimeSpan.FromMilliseconds((int)keepAliveMs);
+				if (typedOptions.TryGetValue(KeepAliveInterval, out var keepAliveIntervalMs)) {
+					settings.ConnectivitySettings.KeepAliveInterval = keepAliveIntervalMs switch {
+						int value when value == -1 => Timeout.InfiniteTimeSpan,
+						int value when value >= 0 => TimeSpan.FromMilliseconds(value),
+						_ => throw new InvalidSettingException($"Invalid KeepAliveInterval: {keepAliveIntervalMs}")
```

Comment:
**SUGGESTION:** Minor, but in RFC is a bit different format: `$"Invalid keepAliveInterval {keepAliveInterval}. Please provide a positive integer, or -1 to disable."`. 
Same comment for code below https://github.com/EventStore/EventStore-Client-Dotnet/pull/107/files#diff-29e93df819e6a1e3940a362a23ca7857372b6098e23b26c18f4f61282c6319bfR184.